processor/src/main/java/org/mapstruct/ap/model/GeneratedType.java

```diff
@@ -118,6 +124,24 @@ public Accessibility getAccessibility() {
         return accessibility;
     }
 
+    public String getComments() {
```

Comment:
I would suggest to statically hold the comment and initialize it only once (```private static final String COMMENT = initComment();```) - to save some CPU cycles... :wink: